contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -3601,3 +3601,10 @@ AS SELECT
 FROM sys.objects so
 WHERE FALSE;
 GRANT SELECT ON sys.sequences TO PUBLIC;
+
+CREATE OR REPLACE VIEW sys.dm_os_sys_info 
+AS SELECT 
+  CAST(ROUND(EXTRACT(EPOCH FROM NOW()) * 1000.0, 0) AS BIGINT) AS ms_ticks, 
```

Comment:
does this server our purpose? how do we track server start time?

Reply:
Yes, we get server start time from pg_postmaster_start_time() below. To get the uptime Proxy will do (ms_ticks -sqlserver_start_time_ms_ticks).